app/admin/RTMediaUploadTermsAdmin.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * RTMediaUploadTermsAdmin.php
+ * This file sets the admin settings and handle feature of Upload Terms at admin side.
+ *
+ * @package rtMedia
+ * @author  Malav Vasita <malav.vasita@rtcamp.com>
+ */
+
+/**
+ * Class for Admin settings regarding Upload Terms.
+ * We will first check if rtmedia-upload-terms plugin is activate before putting this code in action.
+ * RTMediaUploadTermsAdmin class would come in picture if and only if rtmedia-upload-terms plugin is deactivated.
+ */
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
include once is a statement, not function check if we can use required once instead